RecoTracker/TkSeedGenerator/plugins/SeedGeneratorFromProtoTracksEDProducer.cc

```diff
@@ -143,10 +144,8 @@ void SeedGeneratorFromProtoTracksEDProducer::produce(edm::Event& ev, const edm::
             sqrt(proto.momentum().x() * proto.momentum().x() + proto.momentum().y() * proto.momentum().y());
```

Comment:
SA points to `      es.get<TransientRecHitRecord>().get(builderName, ttrhbESH);` a few lines above.
Can this be migrated as well?

Reply:
Sure. I can migrate that module also. I was originally just focusing on the module TrajectorySeedProducer and touched this module as a knock-on because the change to SeedCreator affected it. I'll push a commit soon.

Reply:
After looking at this more, I realized ttrhbESH is not used, so I just deleted it (I assumed there was not some strange side effect to running the EventSetup get function). Note that I just fixed the one thing referenced in the PR comment. I didn't migrate SeedFromProtoTrack which also uses EventSetup in this module. I could if requested, but that is used elsewhere and possibly would be better handled in a separate PR, if in fact that block of code is ever run. I'll migrate more if requested.